compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3037,17 +3044,17 @@ public void visit(BLangRecordLiteral recordLiteral) {
                     }
 
                     if (bField.type.tag == TypeTags.NEVER) {
-                        neverTypedKeys.add(name);
+                        neverTypedKeys.add(unescapedSpreadExprFiledName);
```

Comment:
Can you check the codecov warning please?

Reply:
Sure will check